.github/workflows/upload-and-deploy.yaml

```diff
@@ -313,3 +315,41 @@ jobs:
 
       - name: Deploy sceptre stacks to staging on prod
         run: pipenv run sceptre --var "namespace=staging" launch prod --yes
+
+
+  integration-test-staging:
```

Comment:
Should we have a section for `if: github.ref_name == 'main'` since we'd be only deploying when a feature gets merged into main?

Reply:
Nevermind, it requires `sceptre-deploy-staging` which already has that